pytket/pytket/qasm/qasm.py

```diff
@@ -1054,6 +1062,11 @@ def circuit_to_qasm_io(
                 if param > 1:
                     param = -2 + param
             params = [param]
+        elif optype == OpType.Barrier and hqs_header(header):
```

Comment:
This should only apply if the header is `hqslib1_dev`, not if it is `hqslib1`.

Reply:
Done